src/api/java/baritone/api/utils/BlockOptionalMeta.java

```diff
@@ -218,43 +216,21 @@ private static VanillaPackResources getVanillaServerPack() {
         return null;
     }
 
-    public static LootDataManager getManager() {
-        if (lootTables == null) {
-            MultiPackResourceManager resources = new MultiPackResourceManager(PackType.SERVER_DATA, List.of(getVanillaServerPack()));
-            ReloadableResourceManager resourceManager = new ReloadableResourceManager(PackType.SERVER_DATA);
-            lootTables = new LootDataManager();
-            resourceManager.registerReloadListener(lootTables);
-            try {
-                resourceManager.createReload(new ThreadPerTaskExecutor(Thread::new), new ThreadPerTaskExecutor(Thread::new), CompletableFuture.completedFuture(Unit.INSTANCE), resources.listPacks().toList()).done().get();
-            } catch (Exception exception) {
-                throw new RuntimeException(exception);
-            }
-
-        }
-        return lootTables;
-    }
-
     private static synchronized List<Item> drops(Block b) {
         return drops.computeIfAbsent(b, block -> {
-            ResourceLocation lootTableLocation = block.getLootTable();
-            if (lootTableLocation == BuiltInLootTables.EMPTY) {
+            ResourceLocation lootTableLocation = block.getLootTable().location();
+            if (lootTableLocation.equals(BuiltInLootTables.EMPTY.location())) {
                 return Collections.emptyList();
             } else {
                 List<Item> items = new ArrayList<>();
                 try {
+                    ServerLevel lv2 = ServerLevelStub.fastCreate();
 
-                    getManager().getLootTable(lootTableLocation).getRandomItemsRaw(
-                            new LootContext.Builder(
-                                    new LootParams.Builder(ServerLevelStub.fastCreate())
-                                            .withParameter(LootContextParams.ORIGIN, Vec3.atLowerCornerOf(BlockPos.ZERO))
-                                            .withParameter(LootContextParams.TOOL, ItemStack.EMPTY)
-                                            .withOptionalParameter(LootContextParams.BLOCK_ENTITY, null)
-                                            .withParameter(LootContextParams.BLOCK_STATE, block.defaultBlockState())
-                                            .create(LootContextParamSets.BLOCK)
-                            ).withOptionalRandomSeed(1L)
-                                    .create(null),
-                            stack -> items.add(stack.getItem())
-                    );
+                    LootParams.Builder lv5 = new LootParams.Builder(lv2)
+                        .withParameter(LootContextParams.ORIGIN, Vec3.ZERO)
+                        .withParameter(LootContextParams.BLOCK_STATE, b.defaultBlockState())
+                        .withParameter(LootContextParams.TOOL, new ItemStack(Items.NETHERITE_PICKAXE, 1));
+                    getDrops(block, lv5).stream().map(ItemStack::getItem).forEach(items::add);
```

Comment:
this didn't work. registry is missing LootContext, so needs to be re-built.

Reply:
i put up a gist for how i've initialized these registries for client access: https://gist.github.com/rfresh2/697c3996dbfdfe595c3322f2adfaa4a6 might be possible to simplify this further

Reply:
does it work now?

Reply:
that appears to work. good enough for now. thanks!